Modules/Devices/test/Devices.Application.Tests/Tests/Identities/Commands/StartDeletionProcessAsSupport/HandlerTests.cs

```diff
@@ -0,0 +1,101 @@
+﻿using Backbone.BuildingBlocks.Application.Abstractions.Exceptions;
+using Backbone.BuildingBlocks.Application.Abstractions.Infrastructure.EventBus;
+using Backbone.Modules.Devices.Application.Identities.Commands.StartDeletionProcessAsSupport;
+using Backbone.Modules.Devices.Application.Infrastructure.Persistence.Repository;
+using Backbone.Modules.Devices.Application.IntegrationEvents.Outgoing;
+using Backbone.Modules.Devices.Domain.Entities.Identities;
+using Backbone.UnitTestTools.Extensions;
+using FakeItEasy;
+using FluentAssertions;
+using Xunit;
+using static Backbone.UnitTestTools.Data.TestDataGenerator;
+
+namespace Backbone.Modules.Devices.Application.Tests.Tests.Identities.Commands.StartDeletionProcessAsSupport;
+
+public class HandlerTests
+{
+    [Fact]
+    public async Task Happy_path_as_support()
+    {
+        // Arrange
+        var activeIdentity = TestDataGenerator.CreateIdentityWithOneDevice();
+
+        var mockIdentitiesRepository = A.Fake<IIdentitiesRepository>();
+        var fakeEventBus = A.Fake<IEventBus>();
```

Comment:
Since you don't use this in any way within the test, you should instantiate it in the CreateHandler method, to reduce the noise in the test